src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -91,19 +83,48 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         _TerminalOutputHandlers(str + L"\r\n");
     }
 
+    // Method description:
+    // - helper that prints exception information to the output stream.
+    // Arguments:
+    // - [IMPLICIT] the current exception context
+    void AzureConnection::_WriteCaughtExceptionRecord()
+    {
+        try
+        {
+            throw;

```

Comment:
what the heck

does this just get the current exception if you're already in a `catch` block?

Reply:
YES it’s great